level1/expressions-types-values.md

```diff
@@ -68,7 +68,7 @@ A type is a property of an expression, not of a value. If you have used a "dynam
 
 > What is the type of `1/0`?
 
-*They should be able to reason about this (expressions have types) but could also check it empirically by running the program.*
+*They should be able to reason about this (expressions have types) but could also check it empirically by running the program.  Discuss this, but don't try to correct their answer.  We haven't yet explained methods*
```

Comment:
Introducing types without introducing methods was pretty confusing.  Perhaps we should leave the explanation at categorization of values, and go on to error prevention when we introduce methods?

Reply:
Interesting. I have mostly worked with students who have some prior programming experience so perhaps they have been more willing to use a prior mental model to explore this question. I'm slightly inclined to keep this. I think it is a good example of the difference between types and tags for those students who have the background to understand it. Perhaps we should have a note to instructors explaining that not all students will be able to make progress here?

Reply:
I agree with keeping this for those with prior programming experience.  

We had a group of two students with no experience, one of whom didn't have an IDE to clearly indicate compile time vs runtime errors.  It's not surprising they found this more challenging than a more mixed group.